index.html

```diff
@@ -12,36 +12,38 @@
         <h4>{{ site.short_description}} </h4>
 
         <div class="browsers">
+        <div class="browser-chrome">
```

Comment:
Could additionally add a `.browser` class to each browser as well, and target that in the CSS, instead of each individual `.browser-xxx`